babel.config.js

```diff
@@ -183,6 +183,18 @@ function bool(value) {
   return value && value !== "false" && value !== "0";
 }
 
+// A minimum semver GTE implementation
+// Limitation:
+// - it only supports comparing major and minor version, assuming Node.js will never ship
+//   features in patch release so we will never need to compare a version with "1.2.3"
+//
+// @example
+// semverGte("8.10", "8.9") // true
+// semverGte("8.9", "8.9") // true
+// semverGte("9.0", "8.9") // true
+// semverGte("8.9", "8.10") // false
+`((v,w)=>(v=v.split("."),w=w.split("."),+v[0]>+w[0]||v[0]==w[0]&&+v[1]>=+w[1]))`;
```

Comment:
should this expression be here?

Reply:
It was meant for documentation of a stringified function only. 🤦‍♂️ 

I didn't figure out how to use `@babel/template` so it can import the function defined here, but since we only used it twice, I am good with copying this function so far.